subprojects/model-core/src/test/groovy/org/gradle/internal/reflect/JavaPropertyReflectionUtilTest.groovy

```diff
@@ -355,7 +355,7 @@ class WithProperties {
 
     void setWriteOnly(String s1) { }
 
-    Boolean isSomething() { return null }
+    boolean isSomething() { return null }
```

Comment:
Does this work? This seems odd.

Reply:
You're right, this is a nonsense example. I'll change the implementation to return false. 

This change was originally done as Groovy 4 doesn't support wrapped Booleans as properties, causing this test to fail.